Unix/tests/http/test_http.cpp

```diff
@@ -22,6 +22,8 @@ static MI_Uint16 PORT = ut::getUnittestPortNumber() + 10;
 #define TEST_USERNAME "test"
 #define TEST_PASSWORD "password"
 
+static const MI_Uint32 MAX_HEADER_SIZE = 65536;
```

Comment:
A magic number. I'd much prefer that you set a constant or #define in the source file, then use that here to size the string. That way, if header size is changed again, the unit test fixes itself automatically.

I think that's the "right" fix. But minimally I'd like a comment explaining where 65536 comes from, and what to do if there are problems. But resolving a const int or #define is much nicer to me.

Reply:
Did you not notice that *is* a constant?

I replaced a magic number with a named constant.  In C++ #define is more or less deprecated for constant values because it produces no symbol.  I put it at the top of the test source with a name so it can be changed as needed. 

I don't see how that differs from what you are saying:

".But resolving a const int or #define is much nicer to me."
vs
static const MI_Uint32 MAX_HEADER_SIZE = 65536;

Note that MI_Uint32 is int (ok, unsigned int)

Reply:
You're focusing on the wrong part. I'm not concerned with the left side of the `=`, I'm concerned of the right hand of the `=` sign.

`65536` is a magic number. Where does it come from? Can it be derived from the implementation itself?